### PR TITLE
Readd: "Add info on parsing logs using our Helm chart"

### PR DESCRIPTION
**Pull request by pmech**
This reverts commit fa3f276b9104a477fed5aa739b3f4531b1275221.

### CODE REVIEW

File: content/integrations/platform-integrations/kubernetes.md [hunk 1]
```diff
@@ -71,6 +72,18 @@ helm install stable/fluent-bit --name=humio-agent -f humio-agent.yaml
...
 
+### Parsing Logs
+
+When using the `FluentBitHelmChart` index as shown in the config
+above, Humio will look for the [pod
+label](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/)
+`humio-parser` and use that parser on the log line. This way you can
+control how data is parsed by just configuring your pods. If the
+`humio-parser` label is not set, the
+[`kv`](http://localhost:1313/parsers/built-in-parsers/kv/) parser will
```
Comment (mwl) [inline content/integrations/platform-integrations/kubernetes.md]: Not exactly the best url. Use ``[`kv`]({{< ref "parsers/build-in-parsers/kv" >}})`` notation

File: content/integrations/platform-integrations/kubernetes.md [hunk 2]
```diff
@@ -30,6 +30,7 @@ backend:
     host: "$HOST"
     http_user: "$INGEST_TOKEN"
     http_passwd: "none"
+    index: "FluentBitHelmChart"
```
Comment (mwl) [inline content/integrations/platform-integrations/kubernetes.md]: Isn't this more specific to FluentBit as to HelmChart? I mean, why limit this functionality to Helm Chart when all Fluent Bit users could benefit from it?

File: content/integrations/platform-integrations/kubernetes.md (hunk 1 above)
Comment (pmech) [reply content/integrations/platform-integrations/kubernetes.md]: thanks!

File: content/integrations/platform-integrations/kubernetes.md (hunk 2 above)
Comment (pmech) [reply content/integrations/platform-integrations/kubernetes.md]: It assumes that you have the field `kubernetes.labels.humio-parser` which is rather k8s specific. That is why I narrow it down.

File: content/integrations/platform-integrations/kubernetes.md [hunk 3]
```diff
@@ -80,7 +80,7 @@ label](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/
 `humio-parser` and use that parser on the log line. This way you can
 control how data is parsed by just configuring your pods. If the
 `humio-parser` label is not set, the
-[`kv`](http://localhost:1313/parsers/built-in-parsers/kv/) parser will
+[kv]({{< ref "parsers/built-in-parsers/kv.md" >}}) parser will
```
Comment (mwl) [inline content/integrations/platform-integrations/kubernetes.md]: You can still use the backtics if you want

Comment (mwl) [reply content/integrations/platform-integrations/kubernetes.md]: And no `.md`

Comment (pmech) [reply content/integrations/platform-integrations/kubernetes.md]: It did not look good in the browser. It just looks like script text so you will not notice that it is actually a link

Comment (mwl) [reply content/integrations/platform-integrations/kubernetes.md]: write ``[`kv` parser](…)``. It's about being consistent, not what looks good.

**Pull request merged**